python-frontend/src/main/java/org/sonar/python/types/v2/FunctionType.java

```diff
@@ -98,6 +98,9 @@ public List<ParameterV2> parameters() {
   }
 
   public PythonType returnType() {
+    if (returnType instanceof LazyType lazyType) {
```

Comment:
wrap it to some utils, since we'll needed in a lot of places once we'll introduce lazy types for other places like superclasses